kotlin-demo/build.gradle

```diff
@@ -14,6 +14,9 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+    kotlinOptions {
+        jvmTarget = '1.8'
```

Comment:
Used for the `JVMStatic` annotation, so we can have a nicer call than `MerlinsBeard.Companion.create(this)` for java callers. They can use `MerlinsBeard.create(this)` too 😄 `MerlinsBeard` can only be created from the interface now, the actual implementation is completely hidden from clients.